ngraph/core/include/ngraph/op/normalize_l2.hpp

```diff
@@ -60,5 +57,3 @@ namespace ngraph
         using v0::NormalizeL2;
```

Comment:
Can we remove this using?

Reply:
Yes, we can but it means that **op::** NormalizeL2 will be no longer available and update all of the usages of `op::NormalizeL2` is needed. (Done with commit https://github.com/openvinotoolkit/openvino/pull/6310/commits/e9e92ba8b6eef9fb7513b677650160feba879b78)

Reply:
After removing this using, CI fails because of external usage of **op::** NormalizeL2 by opencv for example. 
So this change has to be reverted.